dubbo-monitor/dubbo-monitor-default/src/main/java/com/alibaba/dubbo/monitor/dubbo/DubboMonitor.java

```diff
@@ -92,6 +93,7 @@ public void send() {
             long maxOutput = numbers[7];
             long maxElapsed = numbers[8];
             long maxConcurrent = numbers[9];
+            String version = getUrl().getParameter(Constants.DEFAULT_PROTOCOL);

```

Comment:
IMO, this variable is named `protocol` may be better.

Reply:
`DEFAULT_PROTOCOL` value is `dubbo`, this code is to get dubbo version but not protocol name. dubbo now upgrade fast, for a team, different project may use different dubbo version(eg 2.5.3, 2.6.0, 2.6.1), IMO, monitor need to know each project's dubbo version.

Reply:
sounds a reasonable data to collect, let me think about it more.